xml-occurrence-parsers/src/main/java/org/gbif/xml/occurrence/parser/identifier/HolyTriplet.java

```diff
@@ -41,6 +41,20 @@ public HolyTriplet(
     this.unitQualifier = unitQualifier;
   }
 
+  public HolyTriplet(String institutionCode, String collectionCode, String catalogNumber) {
+    Preconditions.checkArgument(institutionCode != null && !institutionCode.isEmpty(),
```

Comment:
since Guava is dependency, should we use Strings.isNullOrEmpty?

Reply:
Got rid of commons.lang3